test/plugin.test.js

```diff
@@ -43,7 +43,7 @@ QUnit.module('videojs-errors', {
     this.player = videojs(this.video);
 
     this.player.buffered = function() {
-      return videojs.createTimeRange(0, 0);
+      return (videojs.time && videojs.time.createTimeRanges || videojs.createTimeRange)(0, 0);
```

Comment:
is `videojs.time` new as of videojs 8.x?

Reply:
That's right!